### PR TITLE
Make CompositeSchema not an ISchema

### PR DESCRIPTION
**Pull request by wschin**
Continue removing ISchema one-by-one. It's a part of #1501.

### CODE REVIEW

File: src/Microsoft.ML.Data/DataView/ZipDataView.cs [hunk 1]
```diff
@@ -106,7 +106,7 @@ public RowCursor[] GetRowCursorSet(Func<int, bool> predicate, int n, Random rand
         private sealed class Cursor : RootCursorBase
         {
             private readonly RowCursor[] _cursors;
-            private readonly CompositeSchema _compositeSchema;
+            private readonly ZipBinding _compositeSchema;
```
Comment (Ivanidzo4ka) [inline src/Microsoft.ML.Data/DataView/ZipDataView.cs]: >_compositeSchema [](start = 40, length = 16)

can you change name of variable?

File: src/Microsoft.ML.Data/DataView/CompositeSchema.cs [hunk 2]
```diff
@@ -13,16 +13,16 @@ namespace Microsoft.ML.Data
     /// A convenience class for concatenating several schemas together.
     /// This would be necessary when combining IDataViews through any type of combining operation, for example, zip.
     /// </summary>
-    internal sealed class CompositeSchema : ISchema
+    internal sealed class ZipBinding
```
Comment (Ivanidzo4ka) [inline src/Microsoft.ML.Data/DataView/CompositeSchema.cs]: Shall we rename file as well?

File: src/Microsoft.ML.Data/DataView/CompositeSchema.cs [hunk 3]
```diff
@@ -13,16 +13,16 @@ namespace Microsoft.ML.Data
     /// A convenience class for concatenating several schemas together.
     /// This would be necessary when combining IDataViews through any type of combining operation, for example, zip.
     /// </summary>
```
Comment (artidoro) [inline src/Microsoft.ML.Data/DataView/CompositeSchema.cs]: Maybe we need to change this comment, as it seems to be suggesting that zip is not the only use?

File: src/Microsoft.ML.Data/DataView/CompositeSchema.cs (hunk 2 above)
Comment (wschin) [reply src/Microsoft.ML.Data/DataView/CompositeSchema.cs]: Done.

[the remark this answers is not in the document]
Comment (wschin) [reply src/Microsoft.ML.Data/DataView/CompositeSchema.cs]: It's not a Schema. It takes several input schemas and produce an output schema.

File: src/Microsoft.ML.Data/DataView/ZipDataView.cs (hunk 1 above)
Comment (wschin) [reply src/Microsoft.ML.Data/DataView/ZipDataView.cs]: Sure.

**Pull request merged**